scripts/release.py

```diff
@@ -470,13 +541,30 @@ def main():
     release = Release(rpco_tag, github_token=token)
 
     if not args.existing_release:
-        release.publish_release()
-        release.update_milestones()
-    if not release.pre_release:
+        if not args.do_not_publish_release:
+            release.publish_release()
+        if not args.do_not_update_milestones:
+            release.update_milestones()
+
+    if (not args.do_not_file_docs_issue and
+            not release.pre_release):
         docs_repo = Repo(url=args.docs_repo_url, cache_dir=args.cache_dir,
                          bare=False)
         request_doc_update(token, docs_repo, release)
 
+    # Guess next cycle version number
```

Comment:
There is no guessing. I'm not sure a comment is needed here but if you wish to keep it I think the language should be changed so it doesn't imply release.tag.next_revision() isn't necessarily doing the right thing.